src/server/table.cc

```diff
@@ -30,11 +30,10 @@ void DbTableStats::AddTypeMemoryUsage(unsigned type, int64_t delta) {
 
 DbTableStats& DbTableStats::operator+=(const DbTableStats& o) {
   constexpr size_t kDbSz = sizeof(DbTableStats);
-  static_assert(kDbSz == 184);
+  static_assert(kDbSz == 176);
 
   ADD(inline_keys);
   ADD(obj_memory_usage);
-  ADD(update_value_amount);
```

Comment:
what do you mean PreUpdate() will no longer do anything ?

Reply:
I mean that it will not remove object memory just for `PostUpdate()` to add it back again.
I believe that `update_value_amount` is introduced to learn how much memory is temporarily removed, or to find "leaks" (that we remove and not add). Both will be irrelevant soon, so I think that there's no point in keeping this counter.